app/next-client-app/components/homepage/hero.tsx

```diff
@@ -0,0 +1,31 @@
+import { AnimatedBeamMapping } from "@/components/AnimatedBeamMapping";
+import AnimatedGridPattern from "@/components/magicui/animated-grid-pattern";
+import { cn } from "@/lib/utils";
+
+export default function Hero() {
+  return (
+    <>
+      {/* Hero section */}
+      <div className="flex-col overflow-hidden lg:flex lg:flex-row lg:justify-between items-center pt-10">
+        <div className="flex flex-col lg:space-y-5 space-y-3 lg:w-3/5">
+          <div className="lg:text-5xl text-3xl gap-5 text-center lg:text-left">
+            <h1 className="text-orange-600">OMOP Transforming </h1>
```

Comment:
`Streamlined Data Mapping to OMOP`

Reply:
Are we keeping the "Made easy" part as well?

Reply:
No actually title is: "Streamlining Data Mapping to OMOP"

Reply:
if we keep "Made easy", it will look like this. If we don't keep that part, I need to decide with part need to be highlight be the orange color, right? 

Please also have a look in general everything again, as well. Thanks!

<img width="1363" alt="Screenshot 2024-09-05 at 15 12 58" src="https://github.com/user-attachments/assets/41bf9fa6-dfa1-4b9b-804e-832ca15c8d3a">